test/IRGen/lto_autolink.swift

```diff
@@ -63,6 +63,6 @@ import AutolinkModuleMapLink
 #endif
 
 // UNSUPPORTED: OS=macosx && CPU=arm64
-// UNSUPPORTED: OS=ios && CPU=arm64e
+// UNSUPPORTED: OS=ios && (CPU=arm64e || CPU=armv7k)
```

Comment:
I thought `armv7k` was a watchOS architecture. There are configurations where we compile for `armv7k` and iOS?

Reply:
Ah right, it was watchos-armv7k. I'll update this.